vscode/src/ruby/chruby.ts

```diff
@@ -63,14 +65,39 @@ export class Chruby extends VersionManager {
 
         versionInfo = fallback.rubyVersion;
         rubyUri = fallback.uri;
-      } catch (error: any) {
-        if (error instanceof RubyVersionCancellationError) {
-          // Try to re-activate if the user has configured a fallback during cancellation
-          return this.activate();
-        }
+      }
+    } catch (error: any) {
+      if (error instanceof RubyCancellationError) {
+        // Try to re-activate if the user has configured a fallback during cancellation
+        return this.activate();
+      }
+
+      throw error;
+    }
+
+    // If we couldn't find a Ruby installation, that means there's a `.ruby-version` file, but that Ruby is not
+    // installed. In this case, we fallback to a closest installation of Ruby - preferrably only varying in patch
```

Comment:
```suggestion
    // installed. In this case, we fallback to a closest installation of Ruby - preferably only varying in patch
```
(I'll look into why the spellcheck config missed this)